app/src/main/res/values/component_colors.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <color name="primary_color">@color/oppia_primary_color</color>
```

Comment:
This is incorrect as this color is defined in `color_palette.xml` file.
Have a look at this: https://github.com/oppia/oppia-android/pull/3951#discussion_r736907251

`component_colors` can only have colors from `color_defs.xml` file.

Reply:
@rt4914 I think Ben wanted ```component_colors.xml``` to only reference ```color_palette.xml``` according to this [comment](https://github.com/oppia/oppia-android/pull/3951#discussion_r737740334).

Reply:
Ack.